SingularityBase/src/main/java/com/hubspot/singularity/DeployStyle.java

```diff
@@ -0,0 +1,5 @@
+package com.hubspot.singularity;
+
+public enum DeployStyle {
```

Comment:
Yeah, that's fair, might keep it simpler in the end, move towards all deploys working the same, but 'regular' deploys are just everything at once